tests/e2e/config/jest.performance.config.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
```

Comment:
At a glance, all of these settings are the same as `jest.config.js` so we could just require that here and extend the `reporters` property and add the peformance reporter to it. What do you think?

Reply:
Great call, @alexflorisca! I've applied your suggestion directly.